gradoop-data-integration/src/test/java/org/gradoop/dataintegration/transformation/InvertEdgesTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.dataintegration.transformation;
+
+import org.gradoop.flink.model.GradoopFlinkTestBase;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.model.impl.functions.epgm.ByLabel;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Tests for the invert edge operator.
+ */
+public class InvertEdgesTest extends GradoopFlinkTestBase {
+
+  @Test(expected = NullPointerException.class)
```

Comment:
Please provide a short javadoc at each test because we plan to enable checkstyle for test classes in the future.